admin/forms.py

```diff
@@ -194,3 +196,41 @@ def __init__(
 
     modules = FieldList(FormField(ModuleForm), min_entries=0)
     published = HiddenField('published', default=False)
+
+
+class AboutYouForm(FlaskWTFForm):
+
+    def __init__(self, admin_client, *args, **kwargs):
+        super(AboutYouForm, self).__init__(*args, **kwargs)
+        self.organisation.choices = get_organisation_choices(
```

Comment:
This all looks good but we're not merging it yet because currently getting all orgs is taking too long. We're doing some work in stagecraft to make it more efficient. The first part of this is allowing the list to be filtered by multiple organisation types (in this case department and agency). If this still isn't quick enough we will have further thoughts.

Thinking about this I'm not sure filtering will make it quicker but Michael says we need to do it anyway - users should only be able to choose department of agency.